src/main/java/com/sparrowwallet/sparrow/AppController.java

```diff
@@ -232,6 +235,7 @@ void initializeView() {
                 if(tabs.getTabs().isEmpty()) {
                     Stage tabStage = (Stage)tabs.getScene().getWindow();
                     tabStage.setTitle("Sparrow");
+                    saveTransaction.setVisible(true);
```

Comment:
I think there needs to be an additional line below this:
```
saveTransaction.setDisabled(true);
```

Reply:
Thanks, wasn't sure. Added it now.